File: Java/src/main/java/vn/elca/training/unittesting/dom/Person.java
/*
 * Person
 *
 * Project: Hiccup Framework
 *
 * Copyright 2019 by ELCA Informatik AG
 * Steinstrasse 21, CH-8036 Zurich
 * All rights reserved.
 *
 * This software is the confidential and proprietary information
 * of ELCA Informatik AG ("Confidential Information"). You
 * shall not disclose such "Confidential Information" and shall
 * use it only in accordance with the terms of the license
 * agreement you entered into with ELCA.
 */

package vn.elca.training.unittesting.dom;

import java.util.Date;

/**
 * @author mmn
 *
 */
public class Person {

    private int id;
    private String firstname;
    private String lastname;
    private Date dateOfBirth;
    private double monthlySalary;
    private double annualTax;
    private double inheritTax;
    private double annualTaxPercentage;
    private double savingAccount;
    private double interestAvenue;
    private double annualIncome;
    private double totalTax;

    public Person(int id) {
        this.id = id;
    }

    public Person(int id, String firstname, String lastname, Date dateOfBirth, double monthlySalary) {
        this.id = id;
        this.firstname = firstname;
        this.lastname = lastname;
        this.dateOfBirth = dateOfBirth;
        this.monthlySalary = monthlySalary;
    }

    public Person(int id, String firstname, String lastname, Date dateOfBirth, double monthlySalary,
        double savingAccount) {
        this.id = id;
        this.firstname = firstname;
        this.lastname = lastname;
        this.dateOfBirth = dateOfBirth;
        this.monthlySalary = monthlySalary;
        this.savingAccount = savingAccount;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getFirstname() {
        return firstname;
    }

    public void setFirstname(String firstname) {
        this.firstname = firstname;
    }

    public String getLastname() {
        return lastname;
    }

    public void setLastname(String lastname) {
        this.lastname = lastname;
    }

    public Date getDateOfBirth() {
        return dateOfBirth;
    }

    public void setDateOfBirth(Date dateOfBirth) {
        this.dateOfBirth = dateOfBirth;
    }

    public double getMonthlySalary() {
        return monthlySalary;
    }

    public void setMonthlySalary(double monthlySalary) {
        this.monthlySalary = monthlySalary;
    }

    public double getAnnualTax() {
        return annualTax;
    }

    public void setAnnualTax(double annualTax) {
        this.annualTax = annualTax;
    }

    public double getInheritTax() {
        return inheritTax;
    }

    public void setInheritTax(double inheritTax) {
        this.inheritTax = inheritTax;
    }

    public double getAnnualTaxPercentage() {
        return annualTaxPercentage;
    }

    public void setAnnualTaxPercentage(double annualTaxPercentage) {
        this.annualTaxPercentage = annualTaxPercentage;
    }

    public double getSavingAccount() {
        return savingAccount;
    }

    public void setSavingAccount(double savingAccount) {
        this.savingAccount = savingAccount;
    }

    public double getInterestAvenue() {
        return interestAvenue;
    }

    public void setInterestAvenue(double interestAvenue) {
        this.interestAvenue = interestAvenue;
    }

    public double getAnnualIncome() {
        return annualIncome;
    }

    public void setAnnualIncome(double annualIncome) {
        this.annualIncome = annualIncome;
    }

    public double getTotalTax() {
        return totalTax;
    }

    public void setTotalTax(double totalTax) {
        this.totalTax = totalTax;
    }

}

File: Net/TaxManagement/Dto/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TaxManagement.Dto
{
    public class Person
    {

        private int _id;
        private String _firstName;
        private String _lastName;
        private DateTime _dateOfBirth;
        private double _monthlySalary;
        private double _annualTax;
        private double _inheritTax;
        private double _annualTaxPercentage;
        private double _savingAccount;
        private double _interestAvenue;
        private double _annualIncome;
        private double _totalTax;

        /**
         * Constructor.
         * 
         * @param id
         */
        public Person(int id)
        {
            this._id = id;
        }

        /**
         * Constructor.
         * 
         * @param id
         * @param firstName
         * @param lastName
         * @param dateOfBirth
         * @param monthlySalary
         */
        public Person(int id, String firstName, String lastName, DateTime dateOfBirth,
                double monthlySalary)
        {
            this._id = id;
            this._firstName = firstName;
            this._lastName = lastName;
            this._dateOfBirth = dateOfBirth;
            this._monthlySalary = monthlySalary;
        }

        /**
         * Constructor.
         * 
         * @param id
         * @param firstName
         * @param lastName
         * @param dateOfBirth
         * @param monthlySalary
         */
        public Person(int id, String firstName, String lastName, DateTime dateOfBirth,
                double monthlySalary, double savingAccount)
        {
            this._id = id;
            this._firstName = firstName;
            this._lastName = lastName;
            this._dateOfBirth = dateOfBirth;
            this._monthlySalary = monthlySalary;
            this._savingAccount = savingAccount;
        }

        public String GetFirstName()
        {
            return _firstName;
        }
        public void SetFirstName(String firstName)
        {
            this._firstName = firstName;
        }
        public String GetLastName()
        {
            return _lastName;
        }
        public void SetLastName(String lastName)
        {
            this._lastName = lastName;
        }
        public DateTime GetDateOfBirth()
        {
            return _dateOfBirth;
        }
        public void SetDateOfBirth(DateTime dateOfBirth)
        {
            this._dateOfBirth = dateOfBirth;
        }
        public double GetMonthlySalary()
        {
            return _monthlySalary;
        }
        public void SetMonthlySalary(double monthlySalary)
        {
            this._monthlySalary = monthlySalary;
        }

        public double GetAnnualTax()
        {
            return _annualTax;
        }

        public void SetAnnualTax(double annualTax)
        {
            this._annualTax = annualTax;
        }

        public double GetInheritTax()
        {
            return _inheritTax;
        }

        public void SetInheritTax(double inheritTax)
        {
            this._inheritTax = inheritTax;
        }

        public double GetAnnualTaxPercentage()
        {
            return _annualTaxPercentage;
        }

        public void SetAnnualTaxPercentage(double annualTaxPercentage)
        {
            this._annualTaxPercentage = annualTaxPercentage;
        }

        public double GetSavingAccount()
        {
            return _savingAccount;
        }

        public void SetSavingAccount(double savingAccount)
        {
            this._savingAccount = savingAccount;
        }

        public double GetInterestAvenue()
        {
            return _interestAvenue;
        }

        public void SetInterestAvenue(double interestAvenue)
        {
            this._interestAvenue = interestAvenue;
        }

        public int GetId()
        {
            return _id;
        }

        public void SetId(int id)
        {
            this._id = id;
        }

        public double GetAnnualIncome()
        {
            return _annualIncome;
        }

        public void SetAnnualIncome(double annualIncome)
        {
            this._annualIncome = annualIncome;
        }

        public double GetTotalTax()
        {
            return _totalTax;
        }

        public void SetTotalTax(double totalTax)
        {
            this._totalTax = totalTax;
        }
    }
}
